wix/README.md

```diff
@@ -5,11 +5,13 @@ The WIX toolset is required to build the Windows Installer file, the WIX toolset
 It can be downloaded from http://wixtoolset.org.
 
 ## Build Instructions
+Update the dependencies.wxs file by running ./generate-dependencies.sh (in mingw64 bash shell).
+
 Run candle.exe to build wixobj file from the wxs file:
-candle.exe -arch x64 windows-connector.wxs
+candle.exe -arch x64 windows-connector.wxs dependencies.wxs
```

Comment:
Add example STDOUT. You can copy paste mine:

Windows Installer XML Toolset Compiler version 3.10.2.2516
Copyright (c) Outercurve Foundation. All rights reserved.

windows-connector.wxs
dependencies.wxs